packages/server/src/integrations/utils.ts

```diff
@@ -509,7 +509,7 @@ export async function handleFileResponse(
       type: response.headers["content-type"],
     })
   }
-  presignedUrl = await objectStore.getPresignedUrl(bucket, key, 600)
+  presignedUrl = await objectStore.getPresignedUrl(bucket, key, 60000)
```

Comment:
Was this always incorrect, or are we actually increasing the length of the presigned URL lifetime by 1000x?

Reply:
I believe this is an addition as part of the `feat/automations-and-rest-file-handling` branch rather than something on master - however I think it would probably be good to update the `getPresignedUrl` function to take the duration as an object specifier like `{ validForSeconds: ... }` to make it more obvious what is being specified. Right now its going from 10 minutes to 0.694 days, which does seem a little odd as well - the `Duration` utility from `backend-core` could be helpful here.

Reply:
Yea so I was running into issues with the signed url and just added this to test something, forgot to move it back - I'll switch it to use the Duration utility also

Reply:
So for this i just removed the param altogether and am using the default param on the getPresignedUrl function, which is an hour.